sway-error/src/error.rs

```diff
@@ -2430,6 +2433,18 @@ impl ToDiagnostic for CompileError {
                         },
                 }
             },
+            ConfigurableMissingAbiDecodeInPlace { span } => Diagnostic {
+                reason: Some(Reason::new(code(1), "Configurables need a function named \"abi_decode_in_place\" to be in scope.".to_string())),
+                issue: Issue::error(
+                    source_engine,
+                    span.clone(),
+                    String::new()
+                ),
+                hints: vec![],
+                help: vec![
+                    "Certify you are accessing a compatible version of the \"core\" library.".into()
```

Comment:
Definitely much much better and more helpful then panicking, but since "abi_decode_in_place" is actually never seen by a developer I think they will still be confused and not know how to certify that they are using a compatible version or where exactly to look.

Providing a bit more detailed info, where exactly to look, will definitely help to solve the issue quicker and easier when it happens. E.g:
```suggestion
                    "Function \"abi_decode_in_place\" is defined in the standard library module \"core::codec\".".into(),
                    "Check that you are using a version of the \"core\" standard library that contains this function.".into(),
```

Reply:
Done.